layout/Query.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const _ = require('lodash');
+
+class ServiceConfiguration {
+  constructor() {
+    this._score = [];
+    this._filter = [];
+  }
+
+  filter(view) {
+    this._filter.push(view);
+  }
+
+  score(view) {
+    this._score.push(view);
+  }
+
+  render(vs) {
```

Comment:
ok, but considering that the function is empty, isn't it correct to alert the user that `super.render()` is redundant and doesn't do anything?